app-modules/engagement/src/Notifications/EngagementBatchStartedNotification.php

```diff
@@ -52,24 +52,31 @@ class EngagementBatchStartedNotification extends BaseNotification implements Dat
     use DatabaseChannelTrait;
     use EmailChannelTrait;
 
+    private string $title;
+
     public function __construct(
         public EngagementBatch $engagementBatch,
         public int $jobsToProcess,
-    ) {}
+        public string $deliveryMethod,
+    ) {
+        $this->title = $this->deliveryMethod === 'email'
+                  ? 'Bulk email request is being processed and will be sent shortly.'
+                  : 'Bulk text request is being processed and will be sent shortly.';
+    }
 
     public function toEmail(object $notifiable): MailMessage
     {
         return MailMessage::make()
             ->settings($this->resolveNotificationSetting($notifiable))
-            ->subject('Bulk Engagement processing started')
+            ->subject($this->title)
```

Comment:
Changing the email subject is not mentioned in the scope of the ticket. Please check with Product in the ticket to explain what the subject is now, and whether or not it should also be changed as a part of this ticket? And if so, to what?